gnuradio.lwr

```diff
@@ -53,8 +40,7 @@ satisfy:
   portage: net-wireless/gnuradio
   pkgconfig: gnuradio-runtime
 source: git+https://github.com/gnuradio/gnuradio.git
-gitbranch: maint-3.8
-gitargs: --recursive
+gitbranch: master
```

Comment:
Since we're changing this, the `gnuradio-default` recipe will need this:

```
    gnuradio:
      gitbranch: maint-3.8
```

Reply:
I've been thinking about the problem of installing different versions of GNU Radio recently. They have different dependencies, so overriding the `gitbranch` in the prefix recipes is not enough. Can `depends` be overridden there also?

Reply:
There are gnuradio37 and gnuradio38 recipes, but they're not very useful since OOTs depend on the gnuradio recipe.

Reply:
Yeah that's an issue. It would be cool if the git branch could be passed as a parameter. TBH, I have no answer/solution here. It's one of the many open PyBOMBS issues.

Reply:
@argilo Is this still an issue? Can we merge this?

Reply:
@mbr0wn The reason I haven't merged this yet is that it will break the `gnuradio-default` prefix recipe, which I expect is commonly used to install GNU Radio 3.8. An update to that recipe should probably be included in this pull request.

Reply:
For the sake of having a working pybombs + gnuradio-master, can we just have gnuradio-master not depend on the base gnuradio but instead include all the things it needs?  or make a gnuradio39.lwr.